src/components/startPage/main.css

```diff
@@ -0,0 +1,115 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+    font-family: "roboto-slab,non-serif";
+    font-size: 1vh;
+  }
+  
+  img {
+    width: 27vw;
+    height: 23.5vh;
+  }
+  .poster {
+    width: 20vw;
+    height: 50vh;
+  }
+  .choose {
+    color: #fff;
+    font-size: 8rem;
+  }
+  .start{
+      text-transform: uppercase;
+  }
+  .siteSection {
+    display: flex;
+    position:absolute;
+    bottom:0;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    height: 70vh;;
+    width: 40vw;
+  }
+  .buttonContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 10vh;
+  }
+  .container {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    width: 37.5vw;
+    height: 70vh;
+    right: 0.73vw;
+    top: 15vh;
+  }
+  .selection {
+    background-color: #494949;
+    width: 30.1vw;
+    height: 8vh;
+    overflow: hidden;
+    border-radius: 15px;
+    color: #fff;
+    font-size: 6.4rem;
+  }
+  .start {
+    background-color: #34a639;
+    box-shadow: 0px 4px 10px 5px #797575c4;
+    width: 14vw;
+    height: 8.9vh;
+    overflow: hidden;
+    border-radius: 15px;
+    color: #fff;
+    font-size: 6.4rem;
+  }
+  button {
+    cursor: pointer;
+    position: relative;
+  }
+  a {
+    width: 100%;
+    height: 100%;
+  }
+  button::after {
+    content: " ";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 30.1vw;
+    height: 8vh;
+    text-shadow: 0 0 0 7rem currentColor;
+    box-shadow: inset 2rem 2rem 2rem 0rem silver;
+    opacity: 0;
+  }
+  button:hover::after{
+    opacity: 1;
+  }
+  
```

Comment:
Add some nice hover animations on the buttons. Something subtle.

Reply:
I've deleted pseudoelement i choose to work only with opacity. WIll that be ok?